src/python/pants/backend/python/util_rules/pex.py

```diff
@@ -433,26 +437,28 @@ async def _setup_pex_requirements(
     pip_resolver_args = [*resolve_config.pex_args(), "--resolver-version", "pip-2020-resolver"]
 
     if isinstance(request.requirements, EntireLockfile):
-        lockfile = await Get(LoadedLockfile, LoadedLockfileRequest(request.requirements.lockfile))
+        loaded_lockfile = await Get(
```

Comment:
This rename is good for clarity, but was also necessary to avoid a mypy error, since we reuse this var name below.